src/main/kotlin/dsl/domain/Skill.kt

```diff
@@ -0,0 +1,6 @@
+package dsl.domain
+
+/**
+ * Created by Jaesungchi on 2022.06.03..
+ */
+data class Skill(val name: String)
```

Comment:
아래 코드로 개선해 볼 수 있을까요?
```suggestion
sealed interface Skill {
    val name: String

    data class Hard(override val name: String) : Skill

    data class Soft(override val name: String) : Skill
}
```